src/components/overlays/StartOverlay.module.scss

```diff
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+}
+
+.overlay {
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 50%;
+  -webkit-transform: translateX(-50%);
+  transform: translateX(-50%);
+  background-color: rgba(0, 0, 0, 0.9);
+  text-align: center;
+  flex-wrap: wrap;
+  z-index: 10000000;
+
+  h1 {
+    padding: 2rem 0;
+  }
+
+  form {
+    width: 70vw;
+    // border: 1px solid green;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+
+    div {
```

Comment:
Try to avoid such general HTML tags and use class names instead. It will help you to maintain code in the future